docs/MVS_Outputs.rst

```diff
@@ -155,6 +155,47 @@ To measure this, we propose to compare the energy flows in between the sectors t
 
 ** The content of this section was copied from the conference paper handed in to CIRED 2020**
 
+Self consumption (SC)
+###############################
+
+The self consumption describes the fraction of all renewable energy that is consumed by the system itself.
+
+.. math::
+        SC &=\frac{\sum_{i,j} {E_{generation} (i,j) \cdot w_i} - E_{gridfeedin}(i,j) \cdot w_i}{\sum_{i,j} {E_{generation} (i,j) \cdot w_i}}
+
+        \text{with~} &i \epsilon \text{[PV,Geothermal,…]}
+
+        &j \epsilon \text{[generation assets 1,2,…]}
+
+[1] https://www.sciencedirect.com/science/article/pii/S0960148119315216
+[2] https://www.iip.kit.edu/downloads/McKennaetal_paper_full.pdf
+
+self_sufficiency (SS)
+###############################
+
+The self sufficiency describes the fraction of the total demand that can be covered by renewable generated energy see [1] and [2]
```

Comment:
```suggestion
The self-sufficiency describes the fraction of the total demand that can be covered by renewable generated energy see [1] and [2]
```
Please add hyperlinks to [1] and [2]. Is it only about renewable energy? Or local generation in general?

Reply:
Is SS bound to [0,1]? Ideally, you could also say what the results mean (counts for all KPI):

* If `SS=0`: The demand is not supplied by any local generation
* If `SS=x, x in ]0,1[`: x% of the deman is supplied by local generation
* If `SS=1`: The demand is completely supplied by local generation

What about local generation > demand? What is the result of SS (or SC and so on) then?

Reply:
see my comment in rtd.